packages/frontend-plugin-api/src/definitions/index.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This folder contains definitions for all core APIs.
+//
+// Plugins should rely on these APIs for functionality as much as possible.
+//
+// If you think some API definition is missing, please open an Issue or send a PR!
+
+export * from './auth';
+
+export * from './AlertApi';
+export * from './AppThemeApi';
+export * from './ConfigApi';
+export * from './DiscoveryApi';
+export * from './ErrorApi';
+export * from './FeatureFlagsApi';
+export * from './FetchApi';
+export * from './IdentityApi';
+export * from './OAuthRequestApi';
+export * from './StorageApi';
```

Comment:
Tbh I think it's best if we avoid duplicating all of these definitions for now and stick to the ugly relative imports instead. We're gonna have this duplication around for a pretty long time so there's a pretty big risk that they diverge at some point. I'm thinking it's enough to just do this for the API definitions though, thinking that the `useApi` etc. are stable enough anyway.